packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -282,10 +266,17 @@ export default defineComponent({
 
   &__container {
     z-index: var(--va-modal-container-z-index);
+  }
 
-    &--transition {
-      @include va-modal-transition();
-    }
+  &-enter-from &__container,
+  &-leave-to &__container {
+    opacity: 0;
+    transform: translateY(-30%);
+  }
+
+  &-enter-active &__container,
+  &-leave-active &__container {
+    transition: $modal-opacity-transition, $modal-transform-transition;
```

Comment:
SCSS variables are not included in final build. We use CSS variables, so user can easily redefine them. Please, store this values in _variables.scss. 

(You can see there are a lot of them used in components)